homotopy-graphics/src/style.rs

```diff
@@ -22,3 +33,78 @@ impl Default for VertexShape {
         Self::Circle
     }
 }
+
+pub trait RenderableColor {
+    fn css(&self) -> String;
+    fn tikz(&self) -> String;
+}
+
+pub trait CssStylesheet {
```

Comment:
Could you please move this into the svg module? It can be either in `svg.rs` or another file inside the `svg` directory.

Reply:
I think `svg.rs` is a good place, because I am thinking of moving the SVG generation in there as well.

Reply:
Yeah that makes sense. The `css_stylesheet` method is also used by `SignatureStylesheet` in `app.rs`.

I think, if I do this right, we can remove that struct altogether.

Reply:
That would be great!